src/Console/Command/RetrieveReviews.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Elgentos\Kiyoh\Console\Command;
+
+use Elgentos\Kiyoh\Service\RetrieveKiyohReviews;
+use Magento\Framework\Console\Cli;
+use Magento\Framework\Exception\LocalizedException;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class RetrieveReviews extends Command {
+
+    const COMMAND_NAME = 'elgentos:kiyoh:retrieve';
+    const COMMAND_DESCRIPTION = 'Retrieving Kiyoh reviews through the Publishing API';
+    private RetrieveKiyohReviews $service;
+
+    public function __construct(
+        RetrieveKiyohReviews $service,
+        string $name = null
+    )
+    {
+        parent::__construct($name ?? self::COMMAND_NAME);
+        $this->service = $service;
+    }
+
+    public function configure()
+    {
+        $this->setDescription(self::COMMAND_DESCRIPTION);
+    }
+
+    public function execute(InputInterface $input, OutputInterface $output): int
+    {
+        try {
+            $this->service->execute();
+        } catch (LocalizedException $e) {
+            $output->write($e->getMessage());
```

Comment:
You want to use the error output for errors.

```php
$errorOutput = $output instanceof ConsoleOutputInterface
    ? $output->getErrorOutput()
    : $output;

$errorOutput->write($e->getMessage());
```

Alternatively, in this case you don't want the program to proceed, so you're better off removing the `try-catch` altogether and let the exception handler take care of this. That way you also see a stack trace when you use an output with additional verbosity.